mappings/net/minecraft/entity/ai/brain/task/CatchAndEatEntityTask.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/unmapped/C_vejudmli net/minecraft/entity/ai/task/CatchAndEatEntityTask
+	FIELD f_cwxznlib eatSound Lnet/minecraft/unmapped/C_avavozay;
+	FIELD f_enajckon TONGIE_ANIMATION_DURATION I
+	FIELD f_fkcsyyou EATING_MOVEMENT_FACTOR F
+	FIELD f_iqfdftpz CATCH_ANIMATION_DURATION I
+	FIELD f_miaxvnpq eatAnimationTicks I
+	FIELD f_smpscoss targetEntityPosition Lnet/minecraft/unmapped/C_vgpupfxx;
+	FIELD f_vpcodfwg toungeSound Lnet/minecraft/unmapped/C_avavozay;
+	FIELD f_wdkidpnn EATING_DISTANCE F
+	FIELD f_wqkflrzv TIMEOUT_DURATION I
```

Comment:
`duration` doesn't really fit here, a timeout isn't something that spans time

Reply:
```suggestion
	FIELD f_wqkflrzv EATING_TIMEOUT I
```